app/src/constants/session-storage.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Key used to cache column visiblity in sessionStorage
+ */
+export const SIMS_TELEMETRY_HIDDEN_COLUMNS = 'SIMS_TELEMETRY_HIDDEN_COLUMNS';
+
+/**
+ * Key used to cache column visiblity in sessionStorage
+ */
+export const SIMS_OBSERVATIONS_HIDDEN_COLUMNS = 'SIMS_OBSERVATIONS_HIDDEN_COLUMNS';
+
+/**
+ * Key used to cache the additional user-defined measurement columns added to the observations table.
+ *
+ * Should store a JSON stringified array of `GridColDef` objects.
+ */
+export const SIMS_OBSERVATIONS_MEASUREMENT_COLUMNS = 'SIMS_OBSERVATIONS_MEASUREMENT_COLUMNS';
+
+/**
+ * Get a session storage key which is unique to the provided survey id.
+ *
+ * @param {number} surveyId
+ * @param {string} sessionStorageKey
+ * @return {*}  {string}
+ */
+export const getSurveySessionStorageKey = (surveyId: number, sessionStorageKey: string): string => {
+  return `${sessionStorageKey}_${surveyId}`;
```

Comment:
Nit: I might prefix the survey ID with `SURVEY_`, just to distinguish it from other potential keys.

For instance, if we had a `getProjectSessionStorageKey` method:

```javascript
export const getProjectSessionStorageKey = (projectId: number, sessionStorageKey: string): string => {
  return `${sessionStorageKey}_${projectId}`;
}
const key1 = getSurveySessionStorageKey(mySurveyId, PREFERRED_COLUMN_ORDER);
const key2 = getProjectSessionStorageKey(myProjectId, PREFERRED_COLUMN_ORDER);
// key1 and key2 collide if surveyId and projectId match (plausible) and if I pass the same storage key (not smart, but also plausible)
```

Reply:
Fair point, added.